acme.go

```diff
@@ -66,6 +67,13 @@ const (
 	boulderBuildIDPath    = "/build"
 )
 
+var (
+	// ErrUnsupportedRSABitLen reports whether an unrecognized RSA key size was used
+	ErrUnsupportedRSABitLen = errors.New("unsupported RSA bit length")
```

Comment:
I'd rather them read the errors and fix their calling code. These aren't conditions that seem likely to be resolved programmatically.